sql/item_latents.sql

```diff
@@ -1742,19 +1813,52 @@ INSERT INTO `item_latents` VALUES (16220,10,10,13,114);  -- VIT+10
 
 INSERT INTO `item_latents` VALUES (16238,369,1,13,3);
 
+-- Viator Cape
+INSERT INTO `item_latents` VALUES (16246,2,15,13,267); -- Allied Tags: HP +15
```

Comment:
I think the HP/MP static values aren't real for this latent, needs confirmation. Doesn't follow the pattern of everything else